src/safeds/data/tabular/plotting/_table_plotter.py

```diff
@@ -572,6 +572,8 @@ def moving_average_plot(
                 ylabel=y_name,
             )
             ax.legend()
+            if self._table.get_column(x_name).is_temporal and self._table.get_column(x_name).row_count<9:
+                ax.set_xticks(x_data)  # Set x-ticks to the x data points
```

Comment:
Could you test plotting of data with more than 8 rows?

Reply:
added